src/java/fr/paris/lutece/plugins/subscribe/web/SubscribeApp.java

```diff
@@ -1,218 +1,228 @@
+                SubscriptionFilter filter = new SubscriptionFilter( );
+                filter.setIdSubscriber( user.getName( ) );
+                List<Subscription> listSubscription = subscriptionService.findByFilter( filter );
+                List<SubscriptionDTO> listSubscriptionDto = new ArrayList<SubscriptionDTO>( listSubscription.size( ) );
+                for ( Subscription subscription : listSubscription )
+                {
+                    ISubscriptionProviderService providerService = subscriptionService.getProviderService( subscription
+                            .getSubscriptionProvider( ) );
+                    SubscriptionDTO subscriptionDTO = new SubscriptionDTO( );
+                    subscriptionDTO.setIdSubscription( subscription.getIdSubscription( ) );
+                    subscriptionDTO.setRemovable( providerService.isSubscriptionRemovable( user,
+                            subscription.getSubscriptionKey( ), subscription.getIdSubscribedResource( ) ) );
+                    subscriptionDTO.setUrlModify( providerService.getUrlModifySubscription( user,
+                            subscription.getSubscriptionKey( ), subscription.getIdSubscribedResource( ) ) );
+                    subscriptionDTO.setHtmlSubscription( providerService.getSubscriptionHtmlDescriptionBis( user,
+                            subscription.getSubscriptionKey( ), subscription.getIdSubscribedResource( ),
+                            request.getLocale( ),subscription.getIdSubscribedResource() ) );
+                    listSubscriptionDto.add( subscriptionDTO );
+                }
+
+                Map<String, Object> model = new HashMap<String, Object>( );
+                model.put( MARK_LIST_SUBSCRIPTION_DTO, listSubscriptionDto );
+
+                HtmlTemplate template = AppTemplateService.getTemplate( TEMPLATE_MANAGE_SUBSCRIPTION,
+                        request.getLocale( ), model );
+
+                return template.getHtml( );
+            }
+        }
+        return StringUtils.EMPTY;
+    }
+
+    /**
+     * Get the confirmation message before removing a subscription
+     * @param request The request
+     * @return a XPage
+     * @throws SiteMessageException A SiteMessageException to display the
+     *             confirmation message
+     */
+    @Action( ACTION_CONFIRM_REMOVE_URL )
+    public XPage confirmRemoveSubscription( HttpServletRequest request ) throws SiteMessageException, UnsupportedEncodingException {
+        String strReferer = request.getHeader( PARAMETER_REFERER );
+        UrlItem urlItem = new UrlItem( PATH_PORTAL + getActionUrl( ACTION_DO_REMOVE_URL ) );
+        urlItem.addParameter( PARAMETER_ID_SUBSCRIPTION, request.getParameter( PARAMETER_ID_SUBSCRIPTION ) );
+        Map<String, Object> requestParameters = new HashMap<String, Object>( );
+
+        if(AppPropertiesService.getProperty(ACTIVE_ENCODING_REMOVE_URL).equalsIgnoreCase("true")){
```

Comment:
you can use :
AppPropertiesService.getPropertyBoolean( ... , true)

Reply:
Yes you are right